README.md

```diff
@@ -5,14 +5,47 @@
 
 A WPF Color picker made for [PixiEditor](https://github.com/PixiEditor/PixiEditor). Available to use for anyone, anywhere!
 
-Read about theory behind it on [dev.to](https://dev.to/flabbet/how-does-color-pickers-work-1275)
+![screenshot](https://i.imgur.com/C6m5YWI.png)
 
-![screenshot](https://github.com/PixiEditor/ColorPicker/blob/master/StandardColorPicker.png)
+Read about the theory behind it on [dev.to](https://dev.to/flabbet/how-does-color-pickers-work-1275)
 
 # Controls
 
-Available Standard Color picker `StandardColorPicker` (shown on screenshot above) and Portable Color Picker `PortableColorPicker`
-
-# Example usage
-
-Coming soon!
+There are two controls avaliable: `StandardColorPicker` (shown on the screenshot above) and `PortableColorPicker`. PortableColorPicker is an expandable version of StandardColorPicker.
+
+# Usage
+
+Install the NuGet package, insert a reference to ColorPicker namespace
+```
+<Window ...
+xmlns:colorpicker="clr-namespace:ColorPicker;assembly=ColorPicker"
+...>
+```
+and add either StandardColorPicker or PortableColorPicker controls
+```
+<StackPanel>
+	<colorpicker:StandardColorPicker />
+	<colorpicker:PortableColorPicker />
+</StackPanel>
+```
+Currently selected and secondary colors are stored inside `SelectedColor` and `SecondaryColor` dependency properties.
+
+#Styling
```

Comment:
You forgot space here
```suggestion
# Styling
```

Reply:
And also screenshot is overflowing the text 
![image](https://user-images.githubusercontent.com/25402427/102988001-8ef44380-4513-11eb-9560-845877a2c6a3.png)

Reply:
Oh, for some reason the site I was using to write the readme displayed the markdown correctly and I didn't check on github